components/calendar/index.css

```diff
@@ -19,6 +19,7 @@ governing permissions and limitations under the License.
   --spectrum-calendar-margin-x: 32px;
   --spectrum-calendar-width: calc((var(--spectrum-calendar-day-width) + var(--spectrum-calendar-day-padding) * 2) * 7);
   --spectrum-calendar-button-gap: var(--spectrum-global-dimension-size-40);
+  --spectrum-calendar-title-letter-spacing: var(--spectrum-global-font-letter-spacing-medium);
```

Comment:
This should probably be named `--spectrum-calendar-title-text-letter-spacing` -- everything related to text is prefixed with text.

Also, @NateBaldwinDesign reports this is supposed to be a Detail, so we should use `--spectrum-detail-text-letter-spacing` instead of `--spectrum-global-font-letter-spacing-medium` -- that way we're semantically referring to the Detail style, instead of referring to the value that Detail refers to.